src/embedded_flash/store/mod.rs

```diff
@@ -163,9 +163,11 @@ mod bitfield;
 mod format;
 
 use self::format::{Format, IsReplace};
-#[cfg(feature = "std")]
-use super::BufferStorage;
 use super::{Index, Storage};
+#[cfg(any(test, feature = "ram_storage"))]
+use crate::embedded_flash::BufferStorage;
+#[cfg(any(test, feature = "ram_storage"))]
+use alloc::boxed::Box;
```

Comment:
FYI: Using `test || ram_storage` instead of `std` essentially means we need to set `ram_storage` in fuzzing. But that's fine and anyway won't matter for long.

Reply:
@mingxguo27 FYI

Reply:
Thanks for the fyi. `ram_storage` is already needed in fuzzing, so no problem ^^.

Reply:
Ok good, thanks!